runtime-manager/modules/ROOT/pages/rtm-traffic-inspection.adoc

```diff
@@ -0,0 +1,111 @@
+= Traffic Inspection for Standalone Mules
```

Comment:
This topic doesn't appear in the TOC on the beta site. Where will it live? 

I think we need to take a closer look at the hierarchy of sections in this topic.
```
= Traffic Inspection for Standalone Mules
== Prerequisites
=== Traffic Inspection Proxy
== Mule Installation and Registration Steps
=== Provision the Truststore of the JVM with the Proxy Root CA
=== Install the Mule Runtime
=== Upgrade the Mule Runtime Agent
=== Register the Mule Runtime
=== Check that the Mule Runtime is Connected to the Control Plane
== Notes
```

I would love to figure out how to flatten the hierarchy a little so that the main tasks (H2) appear in the right-hand TOC, if possible.